packages/@eventual/aws-runtime/src/handlers/command-worker.ts

```diff
@@ -19,8 +20,11 @@ import { serviceName } from "../env.js";
  */
 export default createApiGCommandAdaptor({
   commandWorker: createCommandWorker({
-    // the service client, spec, and service url will be created at runtime, using a computed uri from the apigateway request
+    bucketStore: createBucketStore(),
     entityClient: createEntityClient(),
+    // the service client, spec, and service url will be created at runtime, using a computed uri from the apigateway request
+    serviceClient: undefined,
+    serviceSpec: undefined,
```

Comment:
Why is that needed?

Reply:
I made them explicit undefined instead of being able to omit the property so that we are forced to update all locations that use this object when adding a new dependency. In this case it is undefined because the spec is injected at handler invocation time instead of creation time as it is used with the serviceUrl that is taken from the headers instead of the env variable like the rest of the handlers.